src/dotnet-install.ps1

```diff
@@ -145,6 +145,18 @@ function Say-Verbose($str) {
     }
 }
 
+function Say-Text-Separator() {
```

Comment:
I'm unsure on the use case requiring this uncommon function.  The syntax appears correct, though it's difficult to sign off on how it is needed.

Reply:
There is another function, `Say`, which writes whatever you wish there with the prefix "dotnet-install". I do not wish to have this prefix with a blank line, so I decided not to reuse it. I do not want to modify the function `Say` to be able to write a blank line (or in other words, a line with any prefix), as it will involve too wide changes in the code, I will need to pass "dotnet-install" everywhere else.  At the same time I do wish this separator to be written in the same stream as the whole message. So, I copied the logic of the function `Say` to get this.